main.go

```diff
@@ -2,30 +2,48 @@ package main
 
 import "fmt"
 
+var restaurant string = "Jo's coffee shop!"
+
 type List struct {
 	productID   int
```

Comment:
Since the struct can be named `Product`, there is no need to add the `product` prefix to the names of struct fields, so you can just call them `ID`, `Name`, `Price`, etc.